doc/classes/XRFaceTracker.xml

```diff
@@ -0,0 +1,469 @@
+			Mouth corner lip pulls out and down.
+		</constant>
+		<constant name="FT_MOUTH_DIMPLE_RIGHT" value="78" enum="BlendShapeEntry">
+			Right lip corner is pushed backwards.
+		</constant>
+		<constant name="FT_MOUTH_DIMPLE_LEFT" value="79" enum="BlendShapeEntry">
+			Left lip corner is pushed backwards.
+		</constant>
+		<constant name="FT_MOUTH_RAISER_UPPER" value="80" enum="BlendShapeEntry">
+			Raises and slightly pushes out the upper mouth.
+		</constant>
+		<constant name="FT_MOUTH_RAISER_LOWER" value="81" enum="BlendShapeEntry">
+			Raises and slightly pushes out the lower mouth.
+		</constant>
+		<constant name="FT_MOUTH_PRESS_RIGHT" value="82" enum="BlendShapeEntry">
+			Right side lips press and flatten together vertically.
+		</constant>
+		<constant name="FT_MOUTH_PRESS_LEFT" value="83" enum="BlendShapeEntry">
+			Left side lips press and flatten together vertically.
+		</constant>
+		<constant name="FT_MOUTH_TIGHTENER_RIGHT" value="84" enum="BlendShapeEntry">
+			Right side lips squeeze together horizontally.
+		</constant>
+		<constant name="FT_MOUTH_TIGHTENER_LEFT" value="85" enum="BlendShapeEntry">
+			Left side lips squeeze together horizontally.
+		</constant>
+		<constant name="FT_TONGUE_OUT" value="86" enum="BlendShapeEntry">
+			Tongue visibly sticks out of the mouth.
+		</constant>
+		<constant name="FT_TONGUE_UP" value="87" enum="BlendShapeEntry">
+			Tongue points upwards.
+		</constant>
+		<constant name="FT_TONGUE_DOWN" value="88" enum="BlendShapeEntry">
+			Tongue points downwards.
+		</constant>
+		<constant name="FT_TONGUE_RIGHT" value="89" enum="BlendShapeEntry">
+			Tongue points right.
+		</constant>
+		<constant name="FT_TONGUE_LEFT" value="90" enum="BlendShapeEntry">
+			Tongue points left.
+		</constant>
+		<constant name="FT_TONGUE_ROLL" value="91" enum="BlendShapeEntry">
+			Sides of the tongue funnel, creating a roll.
+		</constant>
+		<constant name="FT_TONGUE_BLEND_DOWN" value="92" enum="BlendShapeEntry">
+			Tongue arches up then down inside the mouth.
+		</constant>
+		<constant name="FT_TONGUE_CURL_UP" value="93" enum="BlendShapeEntry">
+			Tongue arches down then up inside the mouth.
+		</constant>
+		<constant name="FT_TONGUE_SQUISH" value="94" enum="BlendShapeEntry">
+			Tongue squishes together and thickens.
+		</constant>
+		<constant name="FT_TONGUE_FLAT" value="95" enum="BlendShapeEntry">
+			Tongue flattens and thins out.
+		</constant>
+		<constant name="FT_TONGUE_TWIST_RIGHT" value="96" enum="BlendShapeEntry">
+			Tongue tip rotates clockwise, with the rest following gradually.
+		</constant>
+		<constant name="FT_TONGUE_TWIST_LEFT" value="97" enum="BlendShapeEntry">
+			Tongue tip rotates counter-clockwise, with the rest following gradually.
+		</constant>
+		<constant name="FT_SOFT_PALATE_CLOSE" value="98" enum="BlendShapeEntry">
+			Inner mouth throat closes.
+		</constant>
+		<constant name="FT_THROAT_SWALLOW" value="99" enum="BlendShapeEntry">
+			The Adam's apple visibly swallows.
```

Comment:
Ah so that's how it's called. This PR is turning into a human biology lesson for me.
```suggestion
			The neck's Adam's apple visibly swallows.
```